web/src/containers/Admin/Deposits/index.js

```diff
@@ -16,6 +16,7 @@ import {
 import { renderRowContent, COLUMNS, SELECT_KEYS } from './utils';
 import { Filters } from './Filters';
 import ValidateDismiss from './ValidateDismiss';
+import { connect } from 'react-redux';
```

Comment:
Just as a convention, it's better to import external libraries at the top of local components and functions.